### PR TITLE
[COS-2303] - Deploy and run temporal in local openline cli

### PR DESCRIPTION
**Pull request by max-openline**
## Changes

What types of changes does your code introduce?  _Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build related changes
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Other (please describe below)

### CODE REVIEW

File: deployment/infra/k8s/events-processing-platform.yaml [hunk 1]
```diff
@@ -100,3 +100,6 @@ spec:
 
             - name: POSTGRES_HOST
               value: "customer-db-postgresql.openline.svc.cluster.local"
+
+            # - name: TEMPORAL_RUN_WORKER
```
Comment (edifirut) [inline deployment/infra/k8s/events-processing-platform.yaml:104]: this shouldn't be true?

**Review by edifirut — CHANGES_REQUESTED** [review]

File: packages/cli/src/lib/dev/temporal-server.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import * as shell from 'shelljs'
+import { getConfig } from '../../config/dev'
+import { deployImage, Yaml } from './deploy'
+import { logTerminal } from '../logs'
+import { log } from 'console'
+
+const config = getConfig()
+const NAMESPACE = config.namespace.name
+const TEMPORAL_SERVER = 'temporal-server'
+const CLI_RAW_REPO = "~/work/openline-cli/" // config.cli.rawRepo
```
Comment (SilviuOpenline) [inline packages/cli/src/lib/dev/temporal-server.ts:10]: why is this path hardcoded? it may point to a local folder of yours. Better provide it via-config

**Review by SilviuOpenline — CHANGES_REQUESTED** [review]

File: deployment/infra/k8s/events-processing-platform.yaml (hunk 1 above)
Comment (max-openline) [reply deployment/infra/k8s/events-processing-platform.yaml:104]: yes, yes it should!

File: packages/cli/src/lib/dev/temporal-server.ts (hunk 2 above)
Comment (max-openline) [reply packages/cli/src/lib/dev/temporal-server.ts:10]: this is leftover i missed, will remove

Comment (SilviuOpenline) [reply packages/cli/src/lib/dev/temporal-server.ts:9]: still not removed

File: packages/cli/src/lib/dev/temporal-server.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    logTerminal('INFO', 'temporal web ui listening on localhost:8233')
+    logTerminal('SUCCESS', 'temporal successfully installed')
+    return true
+}
+
+export function pingTemporalServer(): boolean {
+    // "temporal workflow list" is a command that lists all workflows in the temporal server
+    let podname = shell.exec(`kubectl get pods -n ${NAMESPACE} -l io.kompose.service=temporal-admin-tools -o jsonpath='{.items[0].metadata.name}' `, { silent: true }).stdout
+    return shell.exec(`kubectl exec -it -n ${NAMESPACE} ${podname} -- tctl workflow list`, { silent: true }).code === 0
+    //return shell.exec(`kubectl exec -n ${NAMESPACE} -it $(kubectl get pods -n ${NAMESPACE} -l app=temporal-server -o jsonpath='{.items[0].metadata.name}') -- temporal --ns openline workflow list`, { silent: true }).code === 0
```
Comment (SilviuOpenline) [inline packages/cli/src/lib/dev/temporal-server.ts:34]: remove commented code

**Review by SilviuOpenline — CHANGES_REQUESTED** [review]

**Pull request merged**